### PR TITLE
MAE-51863: HTML Code Style Guide: Change "buttons vs links vs anchors" into a table for easier comparison.

### PR DESCRIPTION
**Pull request by valentinagudelo10**
**JIRA link**: https://vistahl.atlassian.net/browse/MAE-51863

- Change "buttons vs links vs anchors" into a table for easier comparison.

### Screenshot
**Before**
image

**After**
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -166,12 +166,11 @@ Anchors are great for jumping to information farther down a page.
 
 #### Buttons vs.Links vs. Anchors
 ##### The semantic question of link vs button:
- * A button invokes an action: Save, Cancel, Exterminate.
-  * Activated with the `Space` or `Enter` key. Screen reader users may assume this is a form submission and be slightly more trepidacious about activating it than a link.
-  * Identified in screen readers as a button, and included in the list of form controls.
- * A link takes you somewhere (navigates).
-  * Activated with the `Enter` key.
-  * Identified in screen readers as a link, and included in the list of page links.
+| **Buttons** | **Anchors** |
```
Comment (estebanz01) [inline README.md]: I think `Anchors` should be `Links`.

Comment (valentinagudelo10) [reply README.md]: Done!

File: README.md [hunk 2]
```diff
@@ -166,12 +166,11 @@ Anchors are great for jumping to information farther down a page.
...
-  * Activated with the `Space` or `Enter` key. Screen reader users may assume this is a form submission and be slightly more trepidacious about activating it than a link.
-  * Identified in screen readers as a button, and included in the list of form controls.
- * A link takes you somewhere (navigates).
-  * Activated with the `Enter` key.
-  * Identified in screen readers as a link, and included in the list of page links.
+| **Buttons** | **Links** |
+|-|-|
+| A button invokes an action: Save, Cancel, Exterminate. | A link takes you somewhere (navigates). |
+| Activated with the Space or Enter key. Screen reader users may assume this is a form submission and be slightly more trepidacious about activating it than a link. | Activated with the Enter key. |
+| Identified in screen readers as a button, and included in the list of form controls. | Identified in screen readers as a link, and included in the list of page links. |
```
Comment (mhairston) [inline README.md]: Just commenting on content here:

* Add a header cell for the first cell of each data row, so there are row headers and column headers.
* Split "anchors" into 2 columns: one for links (a with href) and one for anchors (no href) and fill in the cells accordingly. (An anchor isn't interactive, but marks an html fragment (should have id) that can be linked to).
* Remove the text about "Screen reader users may assume..." This is not as true as it used to be.
* Add a "Focusable by default?" row with "Yes" for button, "Varies (user Pref)" for link, and "No" for anchor.

Feel free to shorten the wording a bit (it's a table, so it doesn't need to be full sentences)

File: README.md [hunk 3]
```diff
@@ -166,12 +166,11 @@ Anchors are great for jumping to information farther down a page.
...
 
 ##### Considerations when choosing an element:
  * If you style a link to look like a button, a keyboard user will not know to use the `Enter` key to click on it.
```
Comment (mhairston) [inline README.md]: You can remove this item, it's related to the form submission bit above.

**Review by mhairston — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mhairston) [reply README.md]: @estebanz01 I commented below that it needs to be slit into separate columns for "Link" and "Anchor".

**Review by eknelson17 — APPROVED** [review]
I have nothing to add to Mike's review.

File: README.md (hunk 3 above)
Comment (valentinagudelo10) [reply README.md]: Done!

File: README.md (hunk 2 above)
Comment (valentinagudelo10) [reply README.md]: @mhairston I applied the suggests that you give me.

**Review by mhairston — APPROVED** [review]

**Pull request merged**